docs/Contributors_Guide/add_use_case.rst

```diff
@@ -22,32 +38,397 @@ New use cases will be put in the repository under parm/use_cases/model_applicati
+
+In the category sub-directory (parm/use_cases/model_applications/<CATEGORY>),
+each use case should have the following:
+
+* A METplus configuration file named
+  \<MET-TOOL\>_fcst\<FCST\>_obs\<OBS\>_cilmo\<CLIMO\>\<DESCRIPTOR\>.conf where
+
+    * **<MET-TOOL>** is the MET tool that performs the primary statistical
+      analysis, i.e. GridStat or SeriesAnalysis
+
+    * **<FCST>** is the name of the forecast input data source (this can be
+      excluded if no forecast data is used)
+
+    * **<OBS>** is the name of the observation input data source (this can be
+      excluded if no observation data is used)
+
+    * **<CLIMO>** is the optional climotology input data source (this can be
+      excluded if no climatology data is used)
+
+    * **<DESCRIPTION>** is an optional description that can include field
+      category, number of fields, statistical types, and file formats
+
+* 0 or more MET configuration files named <MET-TOOL>Config_<DESCRIPTOR>
+
+
+Document New Use Case
+^^^^^^^^^^^^^^^^^^^^^
+
+In the corresponding documentation category directory
+(**docs**/use_cases/model_applications/<CATEGORY>), add:
+
+* A Python Sphinx Documentation (.py) file with the same name as the METplus
+  configuration file
+
+    * Users are encouraged to copy an existing documentation file and modify it
+      to describe the new use case.
+
+    * Update any references to the .conf file to use the correct name
+
+    * Update the Scientific Objective section to describe the use case
+
+    * Update the description of the input data in the Datasets section
+
+    * Update the list of tools used in the METplus Components section
+
+    * Update the list of run times in the METplus Workflow section
+
+    * Update the list of keywords, referring to :ref:`quick-search` for
+      a list of possible keywords to use (Note: The link text for the
+      keywords must match the actual keyword exactly or it will not
+      show up in the search, i.e. **ASCII2NCToolUseCase** must match
+      https://dtcenter.github.io/METplus/search.html?q=**ASCII2NCToolUseCase**
+
+
+Build the Documentation
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Build the documentation and ensure that the new use case file is
+displayed and the formatting looks correct. The python packages sphinx,
+sphinx-gallery (0.6 or higher), and sphinx_rtd_theme are required to build.
+There is a conda environment called sphinx_env available on some of the NCAR
+development machines that can be used::
+
+    conda activate /home/met_test/.conda/envs/sphinx_env
+
+.. note::
+    If conda is not already in your path, you will have to find it and run it
+    from the full path.
+
+or you can create your own conda environment and install the packages::
+
+    conda create --name sphinx_test python=3.6
+    conda activate sphinx_test
+    conda install sphinx
+    conda install -c conda-forge sphinx-gallery
+    pip install git+https://github.com/ESMCI/sphinx_rtd_theme@version-dropdown-with-fixes
```

Comment:
```suggestion
    pip install git+https://github.com/ESMCI/sphinx_rtd_theme@version-dropdown-with-fixes
```
Thanks for adding this.  I haven't tested it yet, but if it doesn't work, we can fix it later.

Reply:
Sounds good. FYI, you don't have to click the suggested change button if you aren't suggesting a change to the line. You can just add comments in the box.